### DISCUSSION TITLE
Looking for a client server simple example

### DISCUSSION THREAD

**Discussion by dbthebruce** [FI-Mihej/InterProcessPyObjects#7] [Q&A]
This project looks very interesting and I'm pretty sure it will work well for what I'm working on. Thanks for creating it.

I need to have multiple client process send requests to a single server. The server would be persistent. Clients will come and go and will make many requests to the server.

I've spent a few hours modify the example sender / receiver code and I have been unable to get past the one-shot send a single message behavior. I've tried putting a while loop in the receiver function but no matter how I structure it, it only processes the first message.

A very simple example of a persistent receiver would be extremely helpful. 

Thanks, 
Dave

**Comment by FI-Mihej** [comment]
Hi Dave,

Thank you for your interest in the project! I haven’t yet prepared a package for publishing on PyPI that supports communication between a single server process and multiple client processes. However, the code for this functionality is already available on GitHub (see my [Cengal ](https://github.com/FI-Mihej/Cengal) library). This weekend, I plan to work on preparing examples, writing documentation, and finalizing the release of the package on PyPI.

Best regards,
Mike
